[19743] Domain graph & status layout documentation (#205)
eProsima/Fast-DDS-monitor: docs/rst/user_manual/layout.rst

Main Panel
==========
The central window allows multiple tabs for different views.

> Also it is displayed a collapsed menu with the possible problems that have been detected on the DDS entities.

Reviewer: I would add this sentence to the end of the paragraph.

```suggestion
It is als displayed a collapsed menu with the possible problems that have been detected on the DDS entities.
```

> The tab panel allows multiple tabs generation so different views are contemplated in the same window.

Reviewer: This sentence repeats first sentence info

It will display the data charts, so-called *Chartbox*, that the user has configured.
It will display a domain graph where the physical, logical and DDS entities from a domain are represented,
focusing on the connection between endpoints through the topics, and the entities physical inheritance.

.. figure:: /rst/figures/screenshots/main_panel.png
    :align: center

.. _chartbox_layout:

> Chartbox

Reviewer: This section is a bit confusing. Chartbox seems to be the charts displayed in the chart view (it even talks about moving chartbox along the main window). The previous description on how to create a new chartbox seems more appropiate.
There is one only chart view with several chart boxes.

Author: The app is cap now to allow  only one chart view with multiple chartboxes. Eventually, we may support the generation of multiple chart views, so that description will be outdated.
I am keeping it until we fix that.

--------
These windows in the main panel store some *series* or *data configurations* to show a specific data type for
one or several entities in a specified time interval and with different accumulative operations on the data.

> To create a new *Chartbox* go to *Chart View* in the Main Panel default tab.

Reviewer: This instruction creates a new chart view (no more chart views can be created afterwards).

For further details on how to configure a new series refer to :ref:`historic_series` for historic data or
:ref:`dynamic_series` for dynamic data.

.. _domain_graph:

Domain View
-----------
This view in the main panel shows the connections between DataWriters and DataReaders that belong to the same
DDS Domain.
They are represented encapsulated inside their physical entities (see :ref:`entities` relationship), and with a
connection to the topic they are published and subscribed, respectively.

.. figure:: /rst/figures/screenshots/domain_graph.png
    :align: center

> Clicking on any entity displays its detailed information in the :ref:`info_panel`.

Reviewer: ```suggestion
By clicking on any entity its detailed information is displayed in the :ref:`info_panel`.
```

> Right click allows changing the alias of the entity, filter the problems to display only the selected entity's problems
> and, in the case of topics, filter the domain graph by topic, so they are displayed only the entities related to the
> selected topic.

Reviewer: ```suggestion
Right click allows changing the alias of the entity, filter the problems to display only the selected entity's problems
and, in the case of topics, filtering the domain graph by topic so that only the entities related to the selected topic are displayed.
```

.. _problem_summary:

Problem summary
---------------

This section that can be expanded and collapsed displays all the collected problems per entity.
Those problems are related to DataReader samples lost, incompatible QoS between endpoints, or DataWriter deadline missed
counter, for instance.

> The entities that has reported a problem would display a warning or error icon in close to the entity name, based on

Reviewer: ```suggestion
The entities that have reported a problem would display a warning or an error icon close to the entity name, based on
```
